src/compiler/msvc.rs

```diff
@@ -444,6 +445,25 @@ fn compile<T>(creator: &T,
             }
             e @ _ => f_err(e),
         }
+    });
+
+    // If the `-showIncludes` command line option was originally passed we need
+    // to be sure to ship the output from the preprocessor as the actual
+    // result of this compilation.
+    //
+    // Note, though, that when we ran the preprocessor we passed `-E` which
+    // means that the "show includes" business when to stderr. Normally, though,
+    // the compiler emits `-showIncludes` output to stdout. To handle that we
+    // take the stderr of the preprocessor and prepend it to the stdout of the
+    // compilation.
```

Comment:
I think at some point we'll want to do something like this for all compilers to fix #72, but we'll probably have to put the warnings on stderr for compatibility.